crates/starknet-types-core/src/felt/mod.rs

```diff
@@ -21,12 +27,9 @@ pub type BitArrayStore = [u64; 4];
 #[cfg(not(target_pointer_width = "64"))]
 pub type BitArrayStore = [u32; 8];
 
+#[cfg(any(test, feature = "alloc"))]
```

Comment:
why test ?

Reply:
coz alloc::vec::Vec is used to setup tests. But not in the "actual" code